src/lib/replication/repl_dev/solo_repl_dev.cpp

```diff
@@ -21,11 +25,13 @@ SoloReplDev::SoloReplDev(superblk< repl_dev_superblk >&& rd_sb, bool load_existi
                 m_data_journal->register_log_found_cb(bind_this(SoloReplDev::on_log_found, 3));
                 m_is_recovered = true;
             });
+        m_commit_upto = m_rd_sb->durable_commit_lsn;
     } else {
         m_logdev_id = logstore_service().create_new_logdev(flush_mode_t::TIMER);
         m_data_journal = logstore_service().create_new_log_store(m_logdev_id, true /* append_mode */);
         m_rd_sb->logstore_id = m_data_journal->get_store_id();
         m_rd_sb->logdev_id = m_logdev_id;
+        m_rd_sb->checkpoint_lsn = m_rd_sb->last_checkpoint_lsn_1 = m_rd_sb->last_checkpoint_lsn_2 = -1;
```

Comment:
I don't follow this, can you put some comment here why _1 is not -1, _2 is not -2?